api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -169,9 +169,7 @@ public void exportWorkspaces(List<Long> workspaceIds, Boolean backfill) {
       if (!rdrWorkspacesList.isEmpty()) {
         rdrApiProvider.get().exportWorkspaces(rdrWorkspacesList, backfill);
 
-        if (backfill != null && backfill != true) {
-          updateDbRdrExport(RdrEntity.WORKSPACE, workspaceIds);
-        }
+        updateDbRdrExport(RdrEntity.WORKSPACE, workspaceIds);
```

Comment:
We do not want to update the RDR export table when backfilling. The backfill operates on all workspaces. There can be changes that users made to workspaces over the course of the day that should be manually reviewed. If we update the table on backfill we lose sight of user changes as they won't be sent out during the daily cron.

Reply:
Chatted with @calbach we do want to prevent updating the export table on backfill anytime the backfill param is:
```java
if (Optional.ofNullable(backfill).orElse(false) == false) {
  updateDbRdrExport(RdrEntity.WORKSPACE, workspaceIds);
}
```

Reply:
Aha, thanks for explaining this, I figured there was a reason. This is a very nuanced distinction, I added a comment here for future reference, as I definitely will not remember this.